applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -684,6 +965,10 @@ static void stream_released_cb(struct bt_bap_stream *stream)
 				    BT_BAP_EP_STATE_STREAMING)) {
 			le_audio_event_publish(LE_AUDIO_EVT_NOT_STREAMING, stream->conn);
 		}
+	} else if (stream == &headsets[AUDIO_CH_L].source_stream) {
+		LOG_DBG("Left source stream released");
+	} else if (stream == &headsets[AUDIO_CH_R].source_stream) {
```

Comment:
I missed something here: should we also send the LE_AUDIO_EVT_NOT_STREAMING if stream is for source?

Reply:
No, I don't think we should. We currently don't support changing streaming state for bidirectional.